feat: prepare Macaron GitHub Action to publish on GitHub Marketplace (#1259)
oracle/macaron: README.md

![Macaron](./docs/source/assets/macaron.svg)

> [Full Documentation](https://oracle.github.io/macaron/index.html) | [Tutorials](https://oracle.github.io/macaron/pages/tutorials/index.html) | [Videos](https://www.youtube.com/watch?v=ebo0kGKP6bw) | [Papers](#publications) | [Presentations](#presentations) | [Action](https://oracle.github.io/macaron/pages/macaron_action.html)

Reviewer: ```suggestion
[Full Documentation](https://oracle.github.io/macaron/index.html) | [Tutorials](https://oracle.github.io/macaron/pages/tutorials/index.html) | [Videos](https://www.youtube.com/watch?v=ebo0kGKP6bw) | [Papers](#publications) | [Presentations](#presentations) | [Macaron GitHub Action](https://oracle.github.io/macaron/pages/macaron_action.html)
```

**Macaron** is a software supply chain security analysis tool from Oracle Labs focused on verifying the **build integrity** of artifacts and their dependencies. It helps developers, security teams, and researchers ensure that packages are built as expected and have not been tampered with.

> ## Quick Action Usage

Reviewer: ```suggestion
Use Macaron as a GitHub Action
```

> To use the macaron action you can reference it in your workflow.

Reviewer: ```suggestion
To use the Macaron GitHub Action, add the following step to your workflow:
```

```yaml

> - uses: oracle/macaron@v1

Reviewer: ```suggestion
- uses: oracle/macaron@v0.21.0
```
